### PR TITLE
Fix brew osxfuse install instructions.

### PR DESCRIPTION
**Pull request by Stebalien**
Reported by @ejholmes at:

* https://github.com/ipfs/go-ipfs/issues/177#issuecomment-324290935

License: MIT
Signed-off-by: Steven Allen <steven@stebalien.com>

### CODE REVIEW

File: docs/fuse.md [hunk 1]
```diff
@@ -18,7 +18,12 @@ Depending on whether you are using OSX or Linux, follow the proceeding instructi
 
 ## Mac OSX -- OSXFUSE
 
-It has been discovered that versions of `osxfuse` prior to `2.7.0` will cause a kernel panic. For everyone's sake, please upgrade (latest at time of writing is `2.7.4`). The installer can be found at https://osxfuse.github.io/. There is also a homebrew formula (`brew install osxfuse`) but users report best results installing from the official OSXFUSE installer package.
+It has been discovered that versions of `osxfuse` prior to `2.7.0` will cause a kernel panic. For everyone's sake, please upgrade (latest at time of writing is `2.7.4`). The installer can be found at https://osxfuse.github.io/. There is also a homebrew formula using cask but users report best results installing from the official OSXFUSE installer package:
```
Comment (ejholmes) [inline docs/fuse.md]: This is a tad confusing:

> There is also a homebrew formula using cask but users report best results installing from the official OSXFUSE installer package:

which then goes on to show the homebrew install method.

Maybe the sentence should read:

> Users report best results installing from the official OSXFUSE installer package, but there is also a homebrew formula using cask:

**Comment by Stebalien** [comment]
@ejholmes 

Could you test IPFS with osxfuse 3.6.3? I'd like to update the suggested version as the 2.7 series is a bit out of date by this point but don't want to recommend a known broken version.

**Comment by ejholmes** [comment]
Yeah. Ended up not working for me:

**Comment by Stebalien** [comment]
Possibly related: https://github.com/ipfs/go-ipfs/issues/1048

Make sure nothing is currently mounted on either `/ipfs` or `/ipns` (sometimes things don't get properly unmounted when stopping the daemon).

**Comment by whyrusleeping** [comment]
ipfs should probably do some sanity checks before attempting to mount. Should help mitigate these random issues a bit

**Comment by Kubuxu** [comment]
The library we use bazil/fuse is not maintained currently. We might have to migrate to https://github.com/hanwen/go-fuse some time soon. We also really need to extract and clean up the fuseIPFS code into its own module.

**Comment by Stebalien** [comment]
We're going to switch to an entirely new system anyways.

**Pull request closed without merging**